server/svix-server/tests/e2e_attempt.rs

```diff
@@ -64,37 +64,46 @@ async fn test_list_attempted_messages() {
         .await
         .unwrap();
 
-    let list_1: ListResponse<AttemptedMessageOut> = client
-        .get(
-            &format!("api/v1/app/{}/endpoint/{}/msg/", app_id, endp_id_1),
-            StatusCode::OK,
-        )
-        .await
-        .unwrap();
-    let list_2: ListResponse<AttemptedMessageOut> = client
-        .get(
-            &format!("api/v1/app/{}/endpoint/{}/msg/", app_id, endp_id_2),
-            StatusCode::OK,
-        )
-        .await
-        .unwrap();
+    // And wait at most one second for all attempts to be processed
```

Comment:
```suggestion
```
We know what run_with_retries does, it doesn't help, and anyhow, don't assume it'll be a sec.